src/pages/Content/index.js

```diff
@@ -124,7 +134,7 @@ function tryClickUntilExists(selector, interval = 400, maxRetries = 1) {
           reject(new Error('failed to click'));
         }
       }
-      if (status || (retries >= maxRetries)) {
```

Comment:
Once I save the file, it automatically turns from 'if (status || (retries >= maxRetries))' to 'if (status || retries >= maxRetries)'.

Reply:
It's all good, I actually think it is more readable without the extra parentheses.